docs/developer/Code_of_Conduct.md

```diff
@@ -0,0 +1,38 @@
+# OADP Team Code of Conduct
```

Comment:
I'd suggest something more along the lines of "Development Guidelines" as a title here.

Reply:
from today's scrum, I got this impression as well. Would we want a guide like this https://github.com/migtools/oadp-non-admin/blob/master/docs/CONTRIBUTING.md ?

Reply:
yeah +1  for CONTRIBUTING.MD

Reply:
I agree, just we need to make sure there is clear separation between  CONTRIBUTING.MD and Code of Conduct, which are two separate things.

CONTRIBUTING.MD provides guidelines to contributing to the project's codebase, provide technical details, steps for setting up environment, testing standards and so on. They aim to ensure consistency and quality within the codebase.

This PR is about Code of Conduct, which establishes expectations for respectful and collaborative behavior within the community. It is aimed to give positive environment for all contributors and how to interact respectfully and constructively.

Both are important and CONTRIBUTING.MD should be separate PR.

Reply:
@mpryc the content of this PR doesn't really feel like Code of Conduct really -- it's more about the PR review process -- so I think that's actually part of what we generally mean by CONTRIBUTING. Most of what's here is about contributing as an author and contributing as a reviewer.

Reply:
I have broken CONTRIUBTING and CODE_OF_CONDUCT into two pr's.
resolving